runhouse/resources/function.py

```diff
@@ -40,6 +41,7 @@ def __init__(
         """
         self.fn_pointers = fn_pointers
         self.access = access or self.DEFAULT_ACCESS
+        self.func_args = func_args
```

Comment:
can't we extract these ourselves without introducing a new argument? We do this elsewhere using: `func_args = [param.name for param in inspect.signature(fn).parameters.values()]`

Reply:
Made a change so we will not have to introduce a new argument.